package.json

```diff
@@ -63,6 +63,8 @@
 	},
 	"devDependencies": {
 		"@babel/core": "^7.10.5",
+		"@babel/helper-builder-react-jsx": "^7.19.0",
+		"@babel/helper-builder-react-jsx-experimental": "^7.12.11",
```

Comment:
@abaicus since I've added 'addQueryArgs' I get errors saying I need to have '@babel/helper-builder-react-jsx' and '@babel/helper-builder-react-jsx-experimental' when I try to build the project. The same thing happens if I add a simple console.log.
I don't know if this happens only on my end but if I remove those, I can't build the project anymore.

Reply:
@cristian-ungureanu, this seems to be environment specific. I tried using `addQueryArgs` without those dependencies in Neve, and no error was thrown. There can be something out of date in your local environment.

Reply:
Thank you for checking.